src/Command/Ssh/SshKeyCreateCommand.php

```diff
@@ -88,7 +88,7 @@ protected function determineFilename(InputInterface $input, OutputInterface $out
       $this->validateFilename($filename);
     }
     else {
-      $question = new Question('<question>Please enter a filename for your new local SSH key:</question> ', 'id_rsa_acquia');
+      $question = new Question('<question>Please enter a filename for your new local SSH key (default is id_rsa_acquia):</question> ', 'id_rsa_acquia');
```

Comment:
I feel we can still improve this. Suggestions:

* Keep "default" but surround the suggestion by double quotes
* Replace "default" by `Press enter to use "ide_rsa_acquia"`

BTW we also need to change this when creating a Cloud IDE.